xml/osc_building.xml

```diff
@@ -304,6 +306,68 @@ The buildroot was: /var/tmp/build-root/openSUSE_Tumbleweed-x86_64</screen>
   	   in obs://$PROJECT/$REPOSITORY style.
      </para>
    </listitem>
+   <listitem>
+     <para>A predefined milestone <command>OBS-Milestone: milestone</command> 
+     </para>
+     <para>This can predefine a milestone name (eg Beta1) which will be used in cases of a release operation. The candidate will receive it's final tag then.
+     </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+
+ <sect1>
+  <title>Docker Container Specials (&docker;)</title>
```

Comment:
Maybe "Specifics for Docker Container Builds" rather than "Specials"?

"Specials" sounds like this is a restaurant menu.

Reply:
OBS Extensions for Docker builds is maybe better wording?

Reply:
Sounds good to me.